src/index.js

```diff
@@ -13,25 +13,56 @@
 const { wrap: status } = require('@adobe/helix-status');
 const { wrap } = require('@adobe/openwhisk-action-utils');
 const { logger } = require('@adobe/openwhisk-action-logger');
-const { epsagon } = require('@adobe/helix-epsagon');
+const { Response } = require('@adobe/helix-fetch');
 
 const addlogger = require('./addlogger');
 
-async function setupLogger(params) {
-  return {
-    body: await addlogger({
-      email: params.GOOGLE_CLIENT_EMAIL,
-      key: params.GOOGLE_PRIVATE_KEY,
-      service: params.service,
-      token: params.token,
-      project: params.GOOGLE_PROJECT_ID,
-      version: params.version,
-      coralogixkey: params.coralogixkey,
-      coralogixapp: params.coralogixapp,
-      splunkhost: params.splunkhost,
-      splunkauth: params.splunkauth,
-    }),
-  };
+async function setupLogger(request, context) {
+  context.log.info(`Setting up logging: ${request.headers.get('content-type')}`);
+
+  try {
+    let res;
+    if (/^multipart\/formdata|application\/x-www-form-urlencoded/.test(request.headers.get('content-type'))) {
+      context.log.info('Getting parameters from formdata');
+      const data = await request.formData();
```

Comment:
@stefan-guggisberg I'm getting an error here:

```
"message":"Something went wrong TypeError: request.formData is not a function\n at setupLogger (/var/task/index.js:476651:34)\n at /var/task/index.js:15760:12\n at /var/task/index.js:16173:30\n at /var/task/index.js:16129:12\n at /var/task/index.js:39796:13\n at Namespace.run (/var/task/index.js:39775:5)\n at Namespace.runAndReturn (/var/task/index.js:39795:8)\n at wrap (/var/task/index.js:16112:24)\n at /var/task/index.js:16228:23\n at Runtime.lambda [as handler] (/var/task/index.js:3060:28)"
```

Reply:
yes, that's documented: https://github.com/adobe/helix-fetch/blob/main/README.md#about

> `Response.formData()` is not implemented.

Feel free to create an issue if you need it.